appscale/tools/appscale_tools.py

```diff
@@ -818,6 +818,15 @@ def terminate_instances(cls, options):
     except (IOError, AppScaleException):
       # Don't fail if we cannot find the configuration.
       pass
+    except AppControllerException as e:
+      # If we are terminating a cloud infrastructure, we should log and
+      # continue.
+      if (infrastructure in InfrastructureAgentFactory.VALID_AGENTS and
+            options.terminate):
+        AppScaleLogger.warn(e)
```

Comment:
@jeanleonov What happens here is if the tools can't talk to the AppController and they were going to terminate instances, we continue on to terminate those instances. We do still want to run terminate_virtualized_cluster if possible (because it would be a cleaner shutdown)